ios/adapters/epson/NetPrinterEpsonAdapter.m

```diff
@@ -1,112 +1,160 @@
+
 //
 //  NetPrinterEpsonAdapter.m
 //
 //  Created by Till POS on 14/09/21.
 //
 
-
 #import "NetPrinterEpsonAdapter.h"
-#import "EpsonPrinterSDK.h"
-#import "../../utils/NSDataAdditions.h"
 #import "../../utils/EpsonUtils.h"
+#import "../../utils/NSDataAdditions.h"
+#import "EpsonPrinterSDK.h"
 #import <stdlib.h>
 
-@interface NetPrinterEpsonAdapter() <Epos2PtrReceiveDelegate>
-- (void) connectAndSendAux:(NSString *_Nullable)host printRawData: (NSString *_Nullable)text success:(RCTResponseSenderBlock _Nullable )successCallback fail:(RCTResponseSenderBlock _Nullable )errorCallback;
+@interface NetPrinterEpsonAdapter () <Epos2PtrReceiveDelegate>
+- (void)connectAndSendAux:(NSString *_Nullable)host
+             printRawData:(NSString *_Nullable)text
+                  success:(RCTResponseSenderBlock _Nullable)successCallback
+                     fail:(RCTResponseSenderBlock _Nullable)errorCallback
+               printerObj:(Epos2Printer *)printer;
 @end
 
+NSMutableDictionary *printerDictionary;
```

Comment:
probably best to make it static, just in case, not an objective-c expert here

Reply:
Global variable might be easier to export for use by other classes.